app/src/main/java/com/zeapo/pwdstore/git/GitCommandExecutor.kt

```diff
@@ -43,14 +44,24 @@ class GitCommandExecutor(
             message = activity.resources.getString(R.string.git_operation_running),
             length = Snackbar.LENGTH_INDEFINITE,
         )
+        // Count the number of staged files
```

Comment:
Should probably reword it

```suggestion
        // Count the number of uncommitted files
```